app-misc/neofetch/neofetch-2.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Simple information system script"
+HOMEPAGE="https://github.com/dylanaraps/neofetch"
+SRC_URI="https://github.com/dylanaraps/${PN}/archive/${PV}/${P}.tar.gz"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="X"
+
+DEPEND="
+	app-shells/bash:*
```

Comment:
This is most likely incorrect. Slot `:0` installs `bash`, other slots install `bash-X.Y` binaries which your program will likely not use.